substrate/frame/revive/src/storage/meter.rs

```diff
@@ -126,7 +121,7 @@ pub struct RawMeter<T: Config, E, S: State + Default + Debug> {
 	/// limited by the maximum call depth.
 	charges: Vec<Charge<T>>,
 	/// We store the nested state to determine if it has a special limit for sub-call.
-	nested: S,
+	_nested: PhantomData<S>,
 	/// Type parameter only used in impls.
 	_phantom: PhantomData<E>,
```

Comment:
You can remove the nested field.

```suggestion
	_phantom: PhantomData<(E, S)>,
```